#! /bin/sh /usr/share/dpatch/dpatch-run
## udev-rules-use-attrs-not-sysfs.dpatch by Daniel T Chen <crimsun@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix use of deprecated udev syntax (LP: #491346)

@DPATCH@
diff -urNad hplip-3.9.12~/data/rules/56-hpmud_support.rules hplip-3.9.12/data/rules/56-hpmud_support.rules
--- hplip-3.9.12~/data/rules/56-hpmud_support.rules	2009-12-15 16:20:58.000000000 -0500
+++ hplip-3.9.12/data/rules/56-hpmud_support.rules	2010-01-01 16:23:18.000000000 -0500
@@ -7,8 +7,8 @@
 LABEL="pid_test"
 
 # Check for LaserJet products (0x03f0xx17).
-SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="??17", ENV{hp_model}="$sysfs{product}", ENV{hp_test}="yes"
+ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="??17", ENV{hp_model}=ATTRS{product}, ENV{hp_test}="yes"
 
 ENV{hp_test}=="yes", RUN+="bin/sh -c '/usr/bin/hp-mkuri -c &'"
 
-LABEL="hpmud_rules_end"
\ No newline at end of file
+LABEL="hpmud_rules_end"
